anthology/download-proceedings.sh

```diff
@@ -20,7 +20,7 @@ cat $start_urls_file | while read url; do
   fi
   echo "Downloading $url -> data/$acronym"
   [[ ! -d "data/$acronym" ]] && mkdir -p data/$acronym
-  (cd data/$acronym
-  wget -N --no-check-certificate $url/pub/aclpub/proceedings.tgz
+  (cd data/$acronym &&
+  curl -sS --fail -O $url/pub/aclpub/proceedings.tgz &&
```

Comment:
You may need `--insecure` here to replicate wget's `--no-check-certificate` behavior.

Reply:
Can you add it?

This looks good to me.